dask/tests/test_datasets.py

```diff
@@ -40,5 +40,7 @@ def test_no_mimesis():
 def test_deterministic():
     pytest.importorskip("mimesis")
 
+    a = dask.datasets.make_people(seed=123)
     b = dask.datasets.make_people(seed=123)
-    assert b.take(1)[0]["name"] == ("Milda", "Ellis")
+
+    assert a.take(1)[0]["name"] == b.take(1)[0]["name"]
```

Comment:
Made these modifications because it seems like this test would fail as newer versions of `mimesis` impact the random behavior of `mimesis.schema.Field`

Reply:
That's fair, to meet somewhere in the middle I've gone ahead and bumped to 5.3.0, which is sufficient to get the original test passing. I still think it would be better to check the determinism here implicitly to avoid random failures as the version pulled in other builds updates, but I'm okay with switching back to the original test if you think that makes more sense